jsonnet/prometheus-user-workload.jsonnet

```diff
@@ -259,6 +283,39 @@ local sccRole =
                 },
               ],
             },
+            {
+              name: 'kube-rbac-proxy-thanos',
+              image: $._config.imageRepos.kubeRbacProxy + ':' + $._config.versions.kubeRbacProxy,
```

Comment:
We should bump this to https://github.com/brancz/kube-rbac-proxy/releases/tag/v0.8.0